docs/source/_templates/index.html

```diff
@@ -0,0 +1,278 @@
+        ),
+        <span class="cls">Aggregation</span>(
+            input_column=<span class="str">"price"</span>,
+            operation=<span class="cls">Operation</span>.LAST_K(<span class="num">10</span>),
+        )
+    ],
+    online=<span class="kw">True</span>
+)
+            </code></pre>
+        </div>
+
+        <div id="joins" class="code-panel">
+            <div class="code-context">Join multiple feature groups into training sets with guaranteed point-in-time
+                correctness—
+                preventing data leakage automatically.</div>
+            <pre><code>
+<span class="kw">training_set</span> = <span class="cls">Join</span>(
+    left=<span class="cls">EventSource</span>(table=<span class="str">"data.checkouts"</span>, ...),
+    right_parts=[<span class="cls">JoinPart</span>(group_by=purchases_v1)],
+    online=<span class="kw">True</span>
+)
+            </code></pre>
+        </div>
+    </section>
+
+    <!-- Community & Resources -->
+    <section class="resources">
+        <h2>Community & Resources</h2>
+        <div class="resource-grid">
+            <a href="https://medium.com/airbnb-engineering/chronon-a-declarative-feature-engineering-framework-b7b8ce796e04"
+                target="_blank" class="resource-card">
+                <span class="resource-type">Blog Post</span>
+                <h3>Chronon — A Declarative Feature Engineering Framework</h3>
+                <div class="resource-meta">
+                    <span>Airbnb Tech Blog</span>
+                </div>
+            </a>
+            <a href="https://stripe.com/blog/shepherd-how-stripe-adapted-chronon-to-scale-ml-feature-development"
+                target="_blank" class="resource-card">
+                <span class="resource-type">Blog Post</span>
+                <h3>Shepherd: Stripe's next-generation machine learning feature engineering platform</h3>
+                <div class="resource-meta">
+                    <span>Stripe Engineering</span>
+                </div>
+            </a>
+            <a href="https://www.youtube.com/watch?v=0wGLWWZuq6I" target="_blank" class="resource-card">
+                <span class="resource-type">Video</span>
+                <h3>Chronon: Airbnb's Open-Source Data Platform</h3>
+                <div class="resource-meta">
+                    <span>MLOps summit - 2025</span>
+                </div>
+            </a>
+            <a href="https://www.youtube.com/watch?v=Ko186KFDOJ8" target="_blank" class="resource-card">
+                <span class="resource-type">Video</span>
+                <h3>Building Generative Recommenders with Chronon</h3>
+                <div class="resource-meta">
+                    <span>Feature Store summit - 2025</span>
+                </div>
+            </a>
```

Comment:
adding another video  😄 
```suggestion
            <a href="https://www.youtube.com/watch?v=Ko186KFDOJ8" target="_blank" class="resource-card">
                <span class="resource-type">Video</span>
                <h3>Building Generative Recommenders with Chronon</h3>
                <div class="resource-meta">
                    <span>Feature Store summit - 2025</span>
                </div>
            </a>
            <a href="https://video.withairbnb.com/media/t/1_b27ddpnb?st=2725" target="_blank" class="resource-card">
                <span class="resource-type">Video</span>
                <h3>Democratizing High-Performance AI/ML Feature Engineering Through Open Source</h3>
                <div class="resource-meta">
                    <span>AI @ AirBnb - 2025</span>
                </div>
            </a>
```